std/algorithm/sorting.d

```diff
@@ -1062,17 +1062,20 @@ sort(alias less = "a < b", SwapStrategy ss = SwapStrategy.unstable,
 @safe pure nothrow unittest
 {
     int[] array = [ 1, 2, 3, 4 ];
+
     // sort in descending order
-    sort!("a > b")(array);
+    array.sort!("a > b");
     assert(array == [ 4, 3, 2, 1 ]);
+
     // sort in ascending order
-    sort(array);
+    array.sort();
     assert(array == [ 1, 2, 3, 4 ]);
-    // sort with a delegate
-    bool myComp(int x, int y) @safe pure nothrow { return x > y; }
-    sort!(myComp)(array);
-    assert(array == [ 4, 3, 2, 1 ]);
+
+    // sort with a delegate and chain
+    alias myComp = (x, y) => x > y;
```

Comment:
I prefer it not inlined so that it showcases that you can capture lambdas with `alias` like that. Also `myComp` is not bound to a particular element type can be used for sorting other ranges.
BTW, "delegate" is not technically correct, but replacing it with "function template literal" may be too much. Maybe "sort with reusable comparator..."?